lib/rate-limit.js

```diff
@@ -81,5 +81,9 @@ function startDrainer() {
 }
 
 function interval() {
-   return (resetAt * 1000 - Date.now()) / (remaining + 1);;
+   var requestsRemaining = Math.max(1, remaining);
+   // Always underconsume, and leave a minute's worth of requests remaining
+   // just in case our clock drifts from githubs
+   var timeTillReset = Math.max(1, (60 + resetAt) * 1000 - Date.now());
```

Comment:
Yeah it is. https://github.com/iFixit/pulldasher/blob/83410a041d0dc27f0749da43280b3f98481598d4/lib/rate-limit.js#L28